Neos.Fusion.Afx/.styleci.yml

```diff
@@ -0,0 +1 @@
+preset: psr2
```

Comment:
Is this file needed at all?

Reply:
only in case you want to run styleci separately on the split repo I guess

Reply:
removed